ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2162,13 +2165,13 @@ private enum UnityLayers {
       InitializeIntegrators(plugin_, numerics_blueprint);
       BodyProcessor insert_body = body => {
         Log.Info("Inserting " + body.name + "...");
-        ConfigNode body_gravity_model = null;
+        ConfigNode? body_gravity_model = null;
         if (name_to_gravity_model?.TryGetValue(
                 body.name,
                 out body_gravity_model) == true) {
           Log.Info("using custom gravity model");
         }
-        Orbit orbit = unmodified_orbits_.GetValueOrNull(body);
+        Orbit? orbit = unmodified_orbits_!.GetValueOrNull(body);
```

Comment:
Comment that `unmodified_orbits_` has been set in `OnAwake` (we should probably create the dictionary at construction so that `unmodified_orbits_` is not null, and just fill it in `OnAwake`, but we can clean that up later).